### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit includes all relevant
documents to the generated HTML doc for HexDocs.pm and leverages on
latest features of ExDoc.

### CODE REVIEW

File: .formatter.exs [hunk 1]
```diff
@@ -1,4 +1,4 @@
+# Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
-  line_length: 120
```
Comment (AndrewDryga) [inline .formatter.exs:3]: Can you keep the line_length, please?

**Review by AndrewDryga — COMMENTED** [review]
One small thing and LGTM

File: .formatter.exs (hunk 1 above)
Comment (kianmeng) [reply .formatter.exs:3]: Done. Sorry, missed this one.

**Comment by AndrewDryga** [comment]
Thank you ❤️

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**